addons/website_event/static/src/snippets/s_events/000.scss

```diff
@@ -0,0 +1,93 @@
+.s_dynamic {
+    .s_events_event_title {
+        font-weight: $headings-font-weight;
+        line-height: 1;
+    }
+
+    .s_events_event_subtitle {
+        font-size: 1em;
+    }
+
+    &.s_event_event_big_picture {
+        .s_events_event {
+            min-height: 150px;
+
+            figcaption {
+                position: relative;
+                justify-content: center;
+                pointer-events: auto;
+            }
+
+            .s_events_event_cover {
+                min-height: 100%;
+
+                .o_record_cover_container {
+                    top: 0;
+                }
+            }
+
+            .s_events_event_title {
+                @include font-size($h3-font-size);
+                margin-bottom: 0.5em;
+                word-spacing: -0.15em;
```

Comment:
Sure you want that? Every word-spacing and letter-spacing rule we added over the last 7 years have been removed for bug-fixing reasons at some point :smile: